middleware/auth.global.ts

```diff
@@ -0,0 +1,18 @@
+export default defineNuxtRouteMiddleware(async (to, from) => {
+  const app = useNuxtApp()
+  const {localStorage} = window;
+
+  if (!app.$appSettings.auth.enabled) {
+    return;
+  }
+
+  if (to.name !== 'login' && !app.$authToken.token) {
```

Comment:
Thought: We would like to have the store for with data. If it possible, please improve it